config/software/bundler.rb

```diff
@@ -26,7 +26,10 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
-  gem "install bundler" \
-      " --version '#{version}'" \
-      " --no-ri --no-rdoc", env: env
+  if version == "1.10.7.depsolverfix.0"
+    gem "build bundler.gemspec"
+    gem "install bundler-#{version}.gem --no-ri --no-rdoc", env: env
```

Comment:
I forget - is rake a default gem?  I was being paranoid because I didn't want to assume anything but gems existed before installing bundler.